packages/backend-modules/assets/express/pdf.js

```diff
@@ -17,15 +17,17 @@ module.exports = (server) => {
 
     // pick query parameters intended for PDF endpoint
     // (pass along rest as options, later)
-    const { images, download, size, ...options } = req.query
+    const { images, download, format, ...options } = req.query
 
     // build URL to fetch PDF from
+    // TODO: PDF_BASE_URL = https://screenshot.republik.ch/api/pdf
     const fetchUrl = new URL(`${PDF_BASE_URL}/${path}`)
 
     // add params
+    fetchUrl.searchParams.set('url', `${FRONTEND_BASE_URL}/${path}`)
     images && fetchUrl.searchParams.set('images', images)
     download && fetchUrl.searchParams.set('download', download)
-    size && fetchUrl.searchParams.set('size', size)
+    format && fetchUrl.searchParams.set('format', format)
```

Comment:
We should also pass the `version` param: in the frontend we can add the publish date as version, e.g. `?version=2025-01-28T03%3A00Z`, so when an article is re-published, a new PDF will be generated